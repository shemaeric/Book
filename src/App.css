
html, body {
  margin: 0;
  height: 100%;
}
:root {
  --mainNav: #588080;
  --mainBlue: #2a2a72;
  --lightBlue: #009ffd;
  --mainWhite: #d4d0d4;
  --mainDark: #232528;
  --mainYellow: #ffa400;
}
#root{
  height: 100%;
}

* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

body {
  font-family:  "Oswald", sans-serif !important;
  background: var(--mainWhite) !important;
  color: var(--mainDark) !important;
  text-decoration: none;
  height: 100%;
  width: 100%;
}

.text-title {
  font-family: 'Permanent Marker', cursive;
  letter-spacing: 0.3em;
  text-transform: uppercase;
  color: #4b414b;
  width: 10px;
}

.text-blue {
  color: var(--mainBlue);
}

.text-bright {
  color: var(--lightBlue);
}

.nav-logo {
  width: 40%;
  text-decoration: none;
}

.text-center {
  text-align: center;
}

.btn-black {
  background: transparent;
  text-transform: capitalize;
  font-size: 0.8rem !important;
  color: var(--mainDark);
  border-radius: 0 !important;
  border: 0.1rem solid var(--mainDark) !important;

}

.btn-black:hover {
  background: var(--mainDark) !important;
  color: var(--mainWhite) !important;
}

.card {
  height: 25rem !important;
}

.img-container {
  height: 20rem !important;
}

.cart-icon {
  cursor: pointer;
  font-size: 1.4rem;
  color: var(--mainYellow);
}

.login-btn {
  font-family: 'Permanent Marker', cursive;
  background-color: #AF9160;
  border-radius: 1rem;
  font-size: 1.1rem;
  padding: 0.5rem 2rem 0.5rem 2rem;
  color: #d4d0d4;
}

.submit-btn {
  font-family: 'Permanent Marker', cursive;
  background-color: #535546;
  border-radius: 1rem;
  font-size: 1.1rem;
  padding: 0.5rem 2rem 0.5rem 2rem;
  color: #d4d0d4;
}

.sign-btn {
  font-family: 'Permanent Marker', cursive;
  background-color: transparent;
  border: none;
  border-radius: 1rem;
  font-size: 1.1rem;
  padding: 0.5rem 2rem 0.5rem 2rem;
  color: #1b191b;
}

.login-text h5{
  text-align: left;
}


.login-container {
  display: flex;
  width: 100%;
  height: 100%;
  justify-content: center;
  align-items: center;
}

.box-input {
  margin-top: 2.5rem;
  width: 70%;
}

.input-row input{
  margin-top: 0.1rem;
  width: 40rem;
  outline: none;
  border: none;
  border-bottom: #1b191b 1px solid;
  background-color: transparent;
}

.submit-sign {
  margin-top: 2.5rem;
  margin-left: 15rem;
}

.product-container {
  height: 60vh;
  margin-left: 5rem;
}

.card-holder {
  margin: 2rem;
  display: flex;
}
