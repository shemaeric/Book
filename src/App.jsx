import React, { Component } from "react";
import { Switch, Route } from "react-router-dom";
import "./App.css";
import "bootstrap/dist/css/bootstrap.min.css";
import Cart from "./components/Cart/";
import Default from "./components/Default";
import BookList from "./components/BookList";
import BookDetails from "./components/BookDetails";
import Modal from "./components/Modal";
import Signup from "./components/Signup";
import Product from './components/Product/Product';
import Login from "./components/Login";
import { Provider } from "react-redux";
import store from './store';

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <React.Fragment>
        <Switch>
          <Route exact path="/" component={BookList} />
          <Route path="/details" component={BookDetails} />
          <Route path="/cart" component={Cart} />
          <Route path="/login" component={Login} />
          <Route path="/signup" component={Signup} />
          <Route path="/product" component={Product} />
          <Route component={Default} />
        </Switch>
        <Modal />
      </React.Fragment>
      </Provider>
    );
  }
}

export default App;
