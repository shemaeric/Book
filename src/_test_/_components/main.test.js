import React from 'react';
import { shallow, mount } from 'enzyme';
import App from '../../App';

describe('<Routes />', () => {
    test('Should render the Routes', () => {
      const routes = shallow(<App />);
      expect(routes).toMatchSnapshot();
    });
})