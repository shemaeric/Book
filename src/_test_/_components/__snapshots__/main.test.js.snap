// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> Should render the Routes 1`] = `ShallowWrapper {}`;
